### DISCUSSION TITLE
Shirts & Tabards transmog

### DISCUSSION THREAD

**Discussion by BeefSteaker** [chromiecraft/chromiecraft#3318] [Suggestions]
There are Shirts and Tabards of a high enough rarity that they should be transmogable. 

I'd also suggest that all shirts and Tabards should be transmogable irregardless of rarity, but at the very least it should follow the rules for the other slots.

**Comment by Alaeria-Val** [comment]
Reviving the suggestion. Would be nice to be able to use PvP tabards or quest tabards(blood knight) across the account, but at the very least, store tabards should be to be transmogged for all characters on that account.
